llama_index/command_line/command_line.py

```diff
@@ -50,12 +52,53 @@ def handle_download_llama_dataset(
     print(f"Successfully downloaded {llama_dataset_class} to {download_dir}")
 
 
+def handle_question(
+    file_path: Optional[str] = None,
+    question: Optional[str] = None,
```

Comment:
should question be not Optional?

Reply:
actualy both the file path and the question should be mandatory